OpenRA.Game/Support/HttpClientFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Net.Http;
+
+namespace OpenRA.Support
+{
+	public class HttpClientFactory
+	{
+#if !MONO
+		static int maxConnectionPerServer = 20;
```

Comment:
Style nit: make constant.
```suggestion
		const int MaxConnectionPerServer = 20;
```